UnityProject/Assets/Scripts/Systems/Occupations/OccupationCustomEffects/OccupationOnlyGhostMind.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
```

Comment:
add namespace

Reply:
namespace tool is broken on rider, it doesn't show anywhere